swingset/src/main/java/com/nqadmin/swingset/SSComboBox.java

```diff
@@ -822,6 +822,45 @@ public void setSelectedValue(final Integer _value) {
 
 	// }
 
+//	/**
+//	 * Updates the value stored and displayed in the SwingSet component based on
+//	 * getBoundColumnText()
+//	 * <p>
+//	 * Call to this method should be coming from SSCommon and should already have
+//	 * the Component listener removed
+//	 */
+//	@Override
+//	public void updateSSComponent() {
+//		
+//		// TODO Modify this class similar to updateSSComponent() in SSFormattedTextField and only allow JDBC types that convert to Long or Integer
```

Comment:
Discussed a bit in SSComboBox and SSDBComboBox. Goal is to move most/all of the logic here, but didn't want to spend time on Integer vs Long vs M so left very similar logic in the child classes. Can rectify later. See Issue #46.